sources/api/early-boot-config/build.rs

```diff
@@ -4,8 +4,30 @@ use std::env;
 use std::fs::File;
 use std::io::Write;
 use std::path::PathBuf;
+use std::process;
 
 fn main() {
+    // The code below emits `cfg` operators to conditionally compile this program based on the
+    // current variant.  This approach is only meant to be in use for the short term; when the
```

Comment:
This caveat might be better provided with `// TODO - replace this short term approach blah blah` so that it sticks out as something that should be deleted when no longer true.